scripts/automation/drive_client.py

```
@@ -0,0 +1,153 @@
+    """Google Doc を 「Web ページ (zip)」 形式でエクスポートして bytes を返す。"""
+    service = _ensure_drive_service()
+    request = service.files().export_media(
+        fileId=doc_id,
+        mimeType="application/zip",
+    )
+    fh = io.BytesIO()
+    from googleapiclient.http import MediaIoBaseDownload
+    downloader = MediaIoBaseDownload(fh, request)
+    done = False
+    while not done:
+        status, done = downloader.next_chunk()
```

Ruff (0.15.20)

[warning] Unpacked variable `status` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)